modules/rds/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "subnet_ids" {}
+
+variable "env_code" {}
+
+variable "source_security_group" {}
+
+variable "rds_password" {}
+
+variable "vpc_id" {}
```

Comment:
could you please, add a newline at the end of file.

Reply:
fixed.